jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -733,7 +733,14 @@ void janalyzer_parse_optionst::help()
     "Usage:                       Purpose:\n"
     "\n"
     " janalyzer [-?] [-h] [--help] show help\n"
-    " janalyzer class              name of class to be checked\n"
+    " janalyzer class              name of class or JAR file to be checked\n"
+    "                              In the case of a JAR file, if a main class"
+    " can be inferred\n"
+    "                              from --main-class, --function or the JAR"
+    " manifest (checked in\n"
+    "                              this order), the behavior is the same as"
+    " running janalyzer\n"
+    "                              on the corresponding class file."
```

Comment:
:pick: line wrapping is weird

Reply:
Does clang let you indent the (for instance) `" running janalyzer` line?

Reply:
clang isn't a problem, CLion is doing the weird line wrapping thing. What was already in the file doesn't look much nicer but I can try to format the new lines in the same way using vim.